src/main/java/Zap.java

```diff
@@ -0,0 +1,236 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zap {
+    private static List<Task> tasks = new ArrayList<>();
+    public static void main(String[] args) {
+        String logo = """
+              _____   _   _   ____
+             |__  /  / \\ / | |  _ \\
+               / /  / _ \\| | | |_) |
+              / /_ / ___ \\ | |  __/
+             /____/_/   \\_\\_| |_|
+            """;
+
+        System.out.println("Hello from\n" + logo);
+        greeting();
+        processCommands();
+        exit();
+    }
+
+    private static void greeting() {
```

Comment:
Good extraction of main into multiple methods with good method names that clearly describe their functions.